lib/active_merchant/billing/gateways/nuvei.rb

```diff
@@ -123,6 +125,18 @@ def set_initiator_type(post, payment, options)
         post[:isRebilling] = stored_credentials_mode
       end
 
+      def add_account_funding_transaction(post, payment = {}, options = {})
+        return unless options[:is_aft]
+
+        recipient_details = {
+          firstName: options[:cardholder_first_name] || payment.first_name,
+          lastName: options[:cardholder_last_name] || payment.last_name,
+          country: options.dig(:cardholder_country) || options.dig(:billing_address, :country)
```

Comment:
I don't think we want to use custom fields, when we can just use the standard fields. Just use the values we get from `payment` that's given and the `billing_address` options.

Reply:
Hi @naashton, thanks! I did this for two reasons. The first is that the ticket states, "if the GSFs recipientFirstName, recipientLastName, recipientCountry are present, use those values; otherwise, map from the card information." I interpreted this as referring to GSF. Additionally, I saw in the documentation that it's possible to send a refund request with AFT values, and in that refund request, we don't have a payment method.

Reply:
For the second option, I misinterpreted it. The refund only shows us in the response whether the initial purchase was AFT or not.

Reply:
finally I think that make sense only use the firstName and LastName coming from the Payment. Thanks